articles/quickstart/webapp/nodejs-ea/01-login.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login, logout, and profile to a Node.js Express application.
+budicon: 448
+topics:
+  - quickstarts
+  - webapp
+  - login
+  - user profile
+  - logout
+  - nodejs
+contentType: tutorial
+useCase: quickstart
+github:
+  path: 01-Login
+---
+
+<%= include('../../../_includes/_new_app', { hideDownloadSample: true, hideDashboardScreenshot: true }) %>
+<%= include('../../../_includes/_callback_url') %>
+<%= include('../../../_includes/_logout_url' }) %>
+
+## Integrate Auth0
+### Install Dependencies
+Install the following dependencies
+
+- [`express-session`](https://github.com/auth0/express-session) - simple session middleware for Express (any session middleware for express will work).
```

Comment:
Let's switch this to `cookie-session`. Simpler to use and less "gotchas." Similar philosophy to why we're going to skip secret management. A few instances below as well.

Reply:
Also ... it looks like the only two options are `cookie-session` and `express-session` so maybe just call out the ability to use `express-session` but may need additional configuration considerations.